### DISCUSSION TITLE
Adding shaping/via point to existing route

### DISCUSSION THREAD

**Discussion by tweaks90** [devemux86/cruiser#541] [Q&A] [answered]
It used to be that when long pressing on an existing route Cruiser would allow you to add a via/shaping point with a single click and exactly at the point you had clicked on. This doesn't seem to be possible anymore. I do have to separately 'Ok' the new point now.

--> Was this changed deliberately or do I simply need to (re-)enable this via some (new) config option?

Is there a ChangeLog of all implemented changes between the various versions maybe ??

Thanks,
Tom

**Comment by devemux86** [comment, marked as answer]
@tweaks90 

Adding via / shaping points _on the route_ now is done without calculation.
Simply press OK. Previously it required to wait for a new route calculation.

To still allow moving the waypoint before waiting the route calculation,
users need to verify if indeed want the waypoint on the route or move it.

---

There was a long Beta period where all minor changes were discussed and tested:
- https://github.com/devemux86/cruiser/discussions/482

**Comment by tweaks90** [comment]
Right. But how do I know if I managed to actually click on the route. If the point is off the route then you will still need to recalculate the route won't you ? This used to work a bit different I believe, in that the dialog to Ok the new point would only open in case I missed the route.

Cheers

**Reply by devemux86** [reply]
The route menu is very different from the map menu.

See also the discussion here:
- https://github.com/devemux86/cruiser/discussions/476

Route menu|Map menu
-|-
<img src="https://github.com/devemux86/cruiser/assets/3484020/20997a2a-5c4c-4285-b49d-3c072ebdddd6"/>|<img src="https://github.com/devemux86/cruiser/assets/3484020/40a61e68-c0a8-424a-8634-83b80cb754e3"/>

**Reply by tweaks90** [reply]
Ok. Thanks for pointing this out. 
Cheers

**Comment by MotoUKRider** [comment]
Cruiser's map doesn't know about roads except when it does know about roads.  It's a bizarre workflow.

I really think if you want to encourage *road* navigators, then the crosshair feature needs to be optional with automatic snapping-to-road or we'll have to get practising our Bishop Knife Trick :tm: again.
